src/views/SearchWikipedia.vue

```diff
@@ -1,42 +1,105 @@
 <template>
     <div class="view search-wp">
-        <form @submit="onSubmit($event, query)">
-            <h3>Search Wikipedia</h3>
-            <input type="text" placeholder="Search Wikipedia" v-model="query"/>
-            <input type="submit" />
+        <form @submit="onSubmit">
+            <router-link to='Story'><div class="back"/></router-link>
+            <div class="label">{{ $i18n('search-box') }}</div>
+            <input class="query" type="text" :placeholder="[[ $i18n('search-box') ]]" :value="query" @input="update" required/>
+            <input class="submit" type="submit" :value="[[ $i18n('search-text') ]]"/>
```

Comment:
I suggest getting rid of the submit button and search on input.

Reply:
although the design has search button, but I also prefer remove the submit button.